etc/config/c3.defaults.properties

```
@@ -0,0 +1,8 @@
+compilers=c3c
+compilerType=c3c
+defaultCompiler=c3c
+supportsBinary=false
+supportsBinaryObject=false
+supportsExecute=false
+compiler.c3c.name=c3c
+compiler.c3c.exe=/usr/bin/c3c/build/c3c
```

We have had some issues with compiler id clashing between default and amazon envs, this fixes it

```suggestion
compilers=c3cdefault
compilerType=c3c
defaultCompiler=c3cdefault
supportsBinary=false
supportsBinaryObject=false
supportsExecute=false
compiler.c3cdefault.name=c3c
compiler.c3cdefault.exe=/usr/bin/c3c/build/c3c
```